dvc/scm/git.py

```diff
@@ -5,7 +5,7 @@
 import shlex
 
 import yaml
-from funcy import cached_property
+from funcy import cached_property, partial
```

Comment:
:eyes: any reason not `functools.partial`?

Reply:
afaik `funcy.partial` just re-imports `functools.partial`. We use both elsewhere in DVC but I guess we should probably pick one for consistency.

Reply:
doesn't make sense using things via indirect import if they're really the same thing

Reply:
If it's easy to do, sounds like we may need some sort of linter (e.g. `pylint`) to detect indirect imports?